docs/stencil-docs/reference-docs/front-matter-reference.md

```diff
@@ -20,7 +21,7 @@ Front matter defines which store resources are available to be rendered within a
 You can use YAML Front Matter for templates in the `templates/pages/` directory. Injecting objects in the front matter of `templates/pages/page.html` will make the objects available to custom templates.

```

Comment:
All references to "front matter" should be lowercase by default unless at the beginning of a sentence.